interface/main/main_screen.php

```diff
@@ -116,6 +116,10 @@
 if ($GLOBALS['new_tabs_layout']) {
```

Comment:
good point , then maybe better to do:
// For app mode, always use tabs layout
if ((isset($_POST['appChoice'])) && ($_POST['appChoice'] !== '*OpenEMR')) {
    $GLOBALS['new_tabs_layout'] = true;
}
if ($GLOBALS['new_tabs_layout']) {